### PR TITLE
chore(ComicK): update domain

### LINKED ISSUE #7907: ComicK | comick.cc - update presence domain
### Presence name

ComicK

### Description

Currently the ComicK presence is on https://comick.app/
The new website is on https://comick.cc/

The presence doesn't work at all after this change and needs to update the url

### Alternatives

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by veryCrunchy**
resolves #7907

### CODE REVIEW

File: websites/C/ComicK/metadata.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
 		"en": "🚀 ComicK - Fastest Comic Reader",
 		"vi_VN": "🚀 ComicK - Trang đọc truyện tranh nhanh nhất"
 	},
-	"url": "comick.app",
+	"url": "comick.cc",
```
Comment (Bas950) [inline websites/C/ComicK/metadata.json:18]: Regex? They seem to own atleast:
.app
.cc
.ink
.fun

Comment (veryCrunchy) [reply websites/C/ComicK/metadata.json:18]: okay boss 🫡

Comment (veryCrunchy) [reply websites/C/ComicK/metadata.json:18]: However
image

Comment (veryCrunchy) [reply websites/C/ComicK/metadata.json:18]: They only seem to use their .cc domain

Comment (Bas950) [reply websites/C/ComicK/metadata.json:18]: But they still own it, and like all other anime websites will probs change domain again in a few months

Comment (veryCrunchy) [reply websites/C/ComicK/metadata.json:18]: https://www.reddit.com/r/ComicK/comments/qvcgzj/faq_updates_domains/
>Can't seem to access our site, here are some domains we use
>Main Domain-- https://comick.app
>
>Second Domain - - https://comick.ink

No need for a regex if I provide an array of urls right?

Comment (Bas950) [reply websites/C/ComicK/metadata.json:18]: > But they still own it, and like all other anime websites will probs change domain again in a few months

yea sure but keep in mind ^

**Comment by veryCrunchy** [comment]
Smh.. chaos pr again

**Review by Bas950 — APPROVED** [review]

**Review by theusaf — APPROVED** [review]

**Pull request merged**